cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -3267,7 +3267,7 @@ request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
 # TODO - Need to check the license related keywords in the event.
 search = `cs_fortigate` sourcetype="fortigate_event" severity IN ("critical", "high") NOT "*license*" \
-| eval cyences_severity = case(severity=="informational", "info", true(), severity) \
+| eval cyences_severity = "high" \
```

Comment:
case(severity=="informational", "info", severity=="critical", "high", true(), severity)

Reply:
``cs_fortigate` sourcetype="fortigate_event" severity IN ("critical", "high")`
We are filtering the events with only high  and critical severity hence kept it as a high.

Reply:
Make sense